docs/topics/api/abuse.rst

```diff
@@ -58,8 +62,9 @@ to if necessary.
     :>json string|null addon_signature: The add-on signature state.
     :>json string|null addon_summary: The add-on summary in the locale used by the client.
     :>json string|null addon_version: The add-on version string.
-    :>json string|null application: The application used by the client.
-    :>json string|null application_locale: The locale used by the client for the application.
+    :>json string|null app: The application used by the client.
+    :>json string|null appversion: The locale used by the client for the application.
+    :<json string|null lang: The language code of the locale used by the client for the application.
     :>json string|null client_id: The client's hashed telemetry ID.
```

Comment:
I know this isn't a change in this PR, but it's pretty weird to specify all the properties twice because you get back what you submitted (and why do we do that anyway?)

Reply:
Technically the values could differ if we do some post-save treatment on them, or there could be extra stuff returned (which is the case here: `addon` and `reporter` keys are added). It could be more elegantly described in the docs though.